mage_ai/frontend/components/PipelineDetail/UploadPipeline/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { useMemo, useState } from 'react';
+
+import ApiErrorType from '@interfaces/ApiErrorType';
+import Button from '@oracle/elements/Button';
+import FileType from '@interfaces/FileType';
+import FileUploader from '@components/FileUploader';
+import FlexContainer from '@oracle/components/FlexContainer';
+import Panel from '@oracle/components/Panel';
+import ProgressBar from '@oracle/components/ProgressBar';
+import Spacing from '@oracle/elements/Spacing';
+import Table from '@components/shared/Table';
+import Text from '@oracle/elements/Text';
+import { DropZoneStyle, TableStyle } from './index.style';
+import { isEmptyObject } from '@utils/hash';
+import { sortByKey } from '@utils/array';
+import ToggleSwitch from '@oracle/elements/Inputs/ToggleSwitch';
+import { useModal } from '@context/Modal';
+import PopupMenu from '@oracle/components/PopupMenu';
+import { UNIT } from '@oracle/styles/units/spacing';
+import router from 'next/router';
```

Comment:
Please move `import router from 'next/router';` to line 1. We usually group the external imports separately from the internal imports.